### DISCUSSION TITLE
[LOCKED] COOS governance document: Part 2: Key COOS principles -> please go to: https://github.com/linked-statistics/COOS/issues/25

### DISCUSSION THREAD

**Discussion by zoltanvereczkei** [linked-statistics/COOS#37] [Ideas]
I am sorry that it is available as an attached file for the moment. We are still new to the Github platform and it is not easy to insert content here in a tabular format. We are working on it to make the next version available on the platform itself and not in the form of an attached document

Document is here

 |
 |
V

_**NEW version: :) **_
[Part_2_Key_COOS_Principles_v1.1.docx](https://github.com/linked-statistics/COOS/files/6644121/Part_2_Key_COOS_Principles_v1.1.docx)

_Old versions. :(_
[Part_2_Key_COOS_Principles_v1.0.docx](https://github.com/linked-statistics/COOS/files/6644123/Part_2_Key_COOS_Principles_v1.0.docx)

**Comment by FlavioRizzolo** [comment]
Principle 1, Compliance with law... not sure this is really a principle to be stated explicitly...

Principle 5, Ontology is up-to-date, could be misleading since it doesn't say with respect to what: changes to the underlying models might take some time to be reflected in the ontology (that is related to the versioning/change management discussion).

Principle 6, Ontology is easily adaptable, requires more explanation. What do we mean by adaptable?

**Comment by zoltanvereczkei** [comment]
Thanks for the feedback Flavio! It is actually my mistake... I wanted to make it clear that this was kind of an inspiration... if the approach is okay for everyone (statement, rationale, implications, quality aspects) or we want a different approach with this... we used the "Handbook on ontologies" are reference (at least that is where we started). 

I agree that many of these starting principles are not relevant as-it-is. Refinement is needed and if not relevant at all, will be removed completely. Also, I want to highlight the question of statistical relevance... but talk to you soon about this!

**Comment by JALinnerud** [comment]
I assume these are COOS **Design** principles? 
Do we need a change control principle? 
Swapping GSIM for COOS in the GSIM v0.8 design principles (See Annex D here: https://statswiki.unece.org/display/gsim/GSIM+v0.8?preview=/68781467/68714947/GSIM%20Specification%201.docx) we would get: 
Principle 0: Change control.
Statement: COOS has change control i.e. the following principles for designing COOS apply to every revision of COOS.
Rationale:
•	It is recommended that local adaptations of COOS apply the design principles.
Implications:
•	New versions of COOS can be only adopted with an assessment against the design principles. 
•	Mechanisms are needed to monitor the implementation of COOS and adapt it where necessary to meet business needs.

**Comment by zoltanvereczkei** [comment]
Thanks for the comments Jenny! I incorporated this into a new version based on the inputs so far and our discussion at our last meeting. Version 1.1 is now linked to the first post, please check that document. The proposal has basically been rewritten since our last meeting, I hope it reflects more what we want to highlight in the principles.

As we also discussed at our last meeting, the last column (quality dimension) is now also completed. I used the UN Fundamental Principles of Official Statistics as starting point but to be honest, I do not feel its necessity now. But let's discuss this on Monday!

**Comment by zoltanvereczkei** [comment]
### Discussions continue using the Google Docs version of the governance document. Available here:
https://github.com/linked-statistics/COOS/issues/25

Please check the latest posts there for the link to the Google Docs.
